spec/support/selenium_webdriver.rb

```diff
@@ -2,20 +2,21 @@
 
 require "capybara/rails"
 
-Capybara.javascript_driver = :headless_chrome
-
 Capybara.register_driver :headless_chrome do |app|
-  Capybara::Selenium::Driver.load_selenium
-  browser_options = ::Selenium::WebDriver::Chrome::Options.new.tap do |opts|
-    opts.args << "--headless"
-    opts.args << "--disable-gpu"
+  options = ::Selenium::WebDriver::Chrome::Options.new.tap do |opts|
+    opts.add_argument("--headless")
+    opts.add_argument("--disable-gpu")
     # Workaround https://bugs.chromium.org/p/chromedriver/issues/detail?id=2650&q=load&sort=-id&colspec=ID%20Status%20Pri%20Owner%20Summary
-    opts.args << "--disable-site-isolation-trials"
-    opts.args << "--no-sandbox"
-    opts.args << "--window-size=1280,1696"
+    opts.add_argument("--disable-site-isolation-trials")
+    opts.add_argument("--window-size=1920,1080")
   end
-  Capybara::Selenium::Driver.new(app, browser: :chrome, options: browser_options)
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options:)
 end
 
-Capybara.disable_animation = true
 Capybara.default_max_wait_time = 7
+Capybara.disable_animation = true
+Capybara.javascript_driver = :headless_chrome
+
+Capybara::Screenshot.register_driver(:headless_chrome) do |driver, path|
+  driver.browser.save_screenshot(path)
+end
```

Comment:
This is mostly:

1. Setting up capybara-screenshot
2. Updating the syntax
3. Removing 1-2 lines of cruft
4. Re-ordering the code a bit
5. Specifying a slightly larger size to make sure feature tests are getting the full-width website and not a responsive version (e.g., with an unexpected hamburger menu)